### PR TITLE
FMT 10.2 and Spdlog 1.12

### PR DESCRIPTION
**Pull request by ReillyBrogan**
Updates fmt to 10.2.1, spdlog to 1.12.0, and dolphin-emu to 5.0.20347 (as the update was otherwise blocked by the old fmt/spdlog). All revdeps received rebuilds.

### CODE REVIEW

File: packages/e/easyeffects/package.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ builddeps  :
     - zita-convolver-devel
 rundeps    :
     - calf
+    - fmt
```
Comment (EbonJaeger) [inline packages/e/easyeffects/package.yml:44]: Should be after `deepfilternet`

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Just one case of mis-ordered dependencies, other than that, looks good! I am a bit curious as to why `paraview` has so many additions when the package was only bumped.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

File: packages/e/easyeffects/package.yml (hunk 1 above)
Comment (EbonJaeger) [reply packages/e/easyeffects/package.yml:44]: We're gonna worry about that later.

**Comment by ReillyBrogan** [comment]
> Just one case of mis-ordered dependencies, other than that, looks good! I am a bit curious as to why `paraview` has so many additions when the package was only bumped.

I imagine there was a version check around a dep and now that it's been rebuilt that version check is now satisfied when it wasn't before. Although it is really odd since the package was last updated in December, but that might have been before I did the GNOME 45.2 update (which came with a bunch of dep updates).

**Pull request merged**